pages/auth/login.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <v-sheet min-width="300px">
+      <h3 class="text-center">{{ i18n.t("login") }}</h3>
+
+      <v-text-field
+        v-model="name"
+        :label="i18n.t('name')"
+        variant="underlined"
+        color="primary"
+      ></v-text-field>
+
+      <password-input-field
+        v-model="password"
+        :label="i18n.t('password')"
+      ></password-input-field>
+
+      <v-btn
+        variant="tonal"
+        rounded
+        block
+        color="primary"
+        :disabled="!name || !password"
+        @click="onLogin"
+      >
+        {{ i18n.t("login") }}
+      </v-btn>
+    </v-sheet>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Composer } from "vue-i18n";
+import { login } from "~/lib/auth";
+import PasswordInputField from "~/components/ui/PasswordInputField.vue";
+
+const i18n: Composer = useI18n();
+const router = useRouter();
+const localePath = useLocalePath();
+
+useHead({
+  title: i18n.t("login"),
+});
+
+useSeoMeta({
+  title: i18n.t("seo_auth_login_title"),
+  ogTitle: i18n.t("seo_auth_login_title"),
+  description: i18n.t("seo_auth_login_description"),
+  ogDescription: i18n.t("seo_auth_login_description"),
+  ogImage: "/assets/jojomatik.png",
```

Comment:
Is it necessary to define this in every page? Or would it take the one from `app.vue` instead? (the one you removed)

Reply:
What do you mean?

I moved it from `app.vue` to the `index.vue`, because the title and description of the `app.vue` / home page should not be applied to the other pages like login etc.